### PR TITLE
add workshop preparation checklist to be used in issue

### PR DESCRIPTION
**Pull request by wikfeldt**
i.e. the regular checklist we use in issues on coderefinery.org,
see e.g. https://github.com/coderefinery/coderefinery.org/issues/444

### CODE REVIEW

File: workshop-administration.md [hunk 1]
```diff
@@ -125,6 +125,32 @@ For self-organized workshops:
...
+- This checklist can be set up as an issue under 
+  https://github.com/coderefinery/coderefinery.org/ or on another 
+  repository to keep track of the progress
+  ```
+  - [ ] reserve dates
+  - [ ] decide workshop organizer
+  - [ ] (online) prepare Zoom link or (in-person) book lecture room
+  - [ ] announce (twitter, notify-me, mailing lists)
+  - [ ] team of instructors complete
+  - [ ] team of helpers complete
```
Comment (rkdarst) [inline workshop-administration.md:139]: Move this to lower - now we recruit helpers as part of registration?

File: workshop-administration.md [hunk 2]
```diff
@@ -125,6 +125,32 @@ For self-organized workshops:
...
+  - [ ] lessons distributed
+  - [ ] prepare lessons
+  - [ ] create exercise repositories
+  - [ ] (in-person) prepare practicals (coffee/tea, sticky notes, extension cacles)
+  - [ ] (online) Zoom roles distributed
+  - [ ] registration open
+  - [ ] registration closed
+  - [ ] enough pre-survey answers
+  - [ ] install instructions sent
+  - [ ] pre-workshop briefing held
```
Comment (rkdarst) [inline workshop-administration.md:150]:
```suggestion
  - [ ] pre-workshop briefing held, helper training
```

File: workshop-administration.md (hunk 1 above)
Comment (wikfeldt) [reply workshop-administration.md:139]: thanks, but i'm not sure what you mean. Move what to lower where?

**Comment by rkdarst** [comment]
We don't need a complete set of helpers before announcing the
workshop: they can be recruited as part of normal registration, too.
I have usually waited until the weeks before to know a full set of
helpers.

[test reply by email to see if it gets threaded properly]

File: workshop-administration.md (hunk 1 above)
Comment (rkdarst) [reply workshop-administration.md:139]: It didn't thread properly.  I replied by email to the comment above, and it got added to the general conversation.

Comment (wikfeldt) [reply workshop-administration.md]: ah, yeah i wasn't thinking much about the list being chronologically ordered, but it's clearer if the "helpers complete" item is lower. Now applied

**Pull request merged**